### ISSUE TITLE
Screen reader compatibility 

### ISSUE THREAD

**Issue by AndrewH2004** [jellyfin/Swiftfin#963]
### Describe the bug

Hi,

im a visually impaired user of jellyfin who uses a screen reader (voiceover) on iOS and Apple TV.

the initial web wrapper app was accessible and the only reason I’m considering Swiftfin is that somehow on the jellyfin app it doesn’t work with h.265 playback.

however the problem I’m having with jellyfin is that my screen reader doesn’t announce what movie I’m focused on in the library just button, could a tag be added to these buttons with the films title. On iOS I can read the title underneath the button but on Apple TV it’s completely inaccessible along with any details of a title such as synopsis, ratings etc when I select the title.

### Application version

Current 

### Where did you install the app from?

App Store

### Device information

iPad 8th gen, iPhone SE 2, TV 4k 2nd gen

### OS version

iOS 17

### Jellyfin server version

10.8.11

**Comment by AndrewH2004** [comment]
Any update?

**Comment by Pa1yn-dev** [comment]
Pull request submitted to hopefully fix this issue.

**Comment by Pa1yn-dev** [comment]
Pull request has been merged, please close this issue.

**Comment by LePips** [comment]
I consider this a broader effort and would first like to establish some sort of guidelines for us before this is closed

**Comment by Pa1yn-dev** [comment]
> I consider this a broader effort and would first like to establish some sort of guidelines for us before this is closed.

Sounds like a good idea, in order to keep the implementation standard across the package.
